sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_client_async.py

```diff
@@ -740,7 +740,7 @@ async def _client_run_async(self, **kwargs):
             return False
         return True
 
-    async def _message_received_async(self, frame, message):
+    async def _message_received_async(self, delivery_id, message):
```

Comment:
self._delivery_tag = self._amqp_transport.get_message_delivery_tag(message, frame) in message.py you are so right

Reply:
In that case since we do have to pass a frame would we want that frame to be the first one because it looks like when delivery_id=None, delivery_tag=None

Reply:
That sounds like it could be a good option.